scripts/_common.sh

```diff
@@ -5,12 +5,19 @@
 #=================================================
 
 # dependencies used by the app
+
+latest_tag=$(curl -s https://api.github.com/repos/firefly-iii/firefly-iii/releases/latest | grep 'tag_name' | cut -d\" -f4)
+tag="5.5.11"
+
```

Comment:
It will be more secure to use the commit ID corresponding to the TAG : 13b96bb3b60f8a9e894f2d7aba2d3287f143af73